lib/consolidate.js

```diff
@@ -189,36 +189,23 @@ exports.liquid = fromStringRenderer('liquid');
 
 exports.liquid.render = function(str, options, fn){
   return promisify(fn, function (fn) {
-    var engine = requires.liquid || (requires.liquid = require('tinyliquid'));
+    var Liquid = requires.liquid || (requires.liquid = require('liquid-node'));
```

Comment:
tinyliquid is supported right now and dropping that support would be a breaking change. So instead of removing tinyliquid please just add liquid-node by checking which of both is present:
```js
    var engine = requires.liquid;
    if (!engine) {
      try {
        engine = requires.liquid = require('tinyliquid');
      } catch (err) {
        try {
          engine = requires.liquid = require('liquid-node');
        } catch (otherError) {
          throw err;
        }
      }
    }
```